postgres/README.md

```diff
@@ -119,6 +122,30 @@ PostgreSQL default logging is to `stderr` and logs do not include detailed infor
       #log_destination = 'eventlog'
     ```
 
+2. To gather detailed duration metrics and make them searchable in the Datadog interface, they will need to be 
+   configured inline with the statement themselves.  See below for the recommended config, note that counterintuitively 
+   both `log_statement` and `log_duration` options are commented out.
```

Comment:
I can't see the `log_duration` option, is that a typo?

Reply:
True - in the default config the option is commented out, but not showing that line here as well is confusing.